cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.js

```diff
@@ -302,6 +305,11 @@ function LogViewerPreviewController ($scope, $window, LogViewerStore, myPreviewL
     });
   };
 
+  vm.toggleSystemLogs = () => {
+    vm.includeSystemLogs = !vm.includeSystemLogs;
+    startTimeRequest();
```

Comment:
Not that it doesn't have fromOffset at all, it just doesn't have a default fromOffset value. After we fetch the first batch of logs, it gets fromOffset as the `offset` of the last log in that batch. We don't need the fromOffset when we request the first batch of logs right?